LibreNMS/Alert/Transport/Pagerduty.php

```diff
@@ -42,9 +42,12 @@ public function deliverAlert($obj, $opts)
         } else {
             $protocol['event_type'] = 'trigger';
         }
+        $details = array();
         foreach ($obj['faults'] as $fault => $data) {
-            $protocol['details'][] = $data['string'];
+            $details = $data;
```

Comment:
You overwrite $details each loop, is that intended or is the loop unnecessary?

Reply:
The loop is unnecessary I've only kept it because it was previously written that way.  This could further be shortened to
$protocol['details'] = $obj['faults'][0];
unset($protocol['details']['string']);

Reply:
Whew ok so looking at this a little more this morning I think the original foreach was there due to the way the foreach loops at lines 497 and 524 are written.  $i++; should be at the end of each of those loops rather than the front, yes?

So rather than point at a specific array index, we can just grab the last element.  That will make this work now and into the future should the previously mentioned loop ever get adjusted.  Recommitting...